Makefile

```diff
@@ -4,8 +4,8 @@
 #
 
 #Magic hackery to allow shell envs to be set in Makefile
-IGNORE := $(shell bash -c "source set_env.sh; env | sed 's/=/:=/' | sed 's/^/export /' > makeenv")
-include makeenv
+#IGNORE := $(shell bash -c "source set_env.sh; env | sed 's/=/:=/' | sed 's/^/export /' > makeenv")
+#include makeenv
```

Comment:
Perhaps remove entirely instead of commenting out? It'll be in the git history if we ever want to find it again :).